src/Illuminate/Console/ConfirmableTrait.php

```diff
@@ -15,6 +15,10 @@
      */
     public function confirmToProceed($warning = 'Application In Production!', $callback = null)
     {
+        if (! $this->input->isInteractive()) {
+            return true;
```

Comment:
`--non-interactive --force` already has that behaviour. But imho, it's just a warning, and non-interactive is run with the intent to run that.